mysite/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==4.1.2
+pre-commit==2.20.0
+markdown==1.0.1
```

Comment:
What about taggit it and other requirements. This PR should add only the markdown requirement. Add others and commit to master directly.